substrate-node/pallets/pallet-tfgrid/src/lib.rs

```diff
@@ -537,8 +537,7 @@ pub mod pallet {
         InvalidZosVersion,
         FarmingPolicyExpired,
 
-        InvalidHRUInput,
```

Comment:
Keeping old errors and adding new ones at the end of the enum variants would be safer.

Reply:
Good remark
According to here https://substrate.stackexchange.com/questions/9830/removing-an-error-member-with-palleterror it can be done but would require to bump the spec_version
I will keep it simple and follow your suggestion